### ISSUE TITLE
Feature request: Duotone regular and Duotone light sets

### ISSUE THREAD

**Issue by mikkelsiggaard** [FortAwesome/Font-Awesome#15320]
**Is your feature request related to a problem? Please describe.**
Currently, if duotone icons is selected, the default line style is "Solid". 

**Describe the solution you'd like**
It would be great to have to option of selecting either "Solid", "Regular" or "Light" line style when duotone is also selected. 

**Describe alternatives you've considered**
For offline use it would be rather simple to fix in Illustrator, but on web there is no preferable alternative. 

**Additional context**

**Feature request checklist**
- [x] This is a single feature (i.e. not a re-write of all of Font Awesome)
- [x] The title starts with "Feature request: " and is followed by a clear feature name (Ex: `Feature request: moar cowbell`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome-Pro/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

If I've correctly understood this request, you are asking for duotone styles for regular and light icons, too.

So, if I'm correct, this is a request to build ~ 3.288 (1644 * 2, at the moment I'm writing) new icons, and it looks like *almost* a complete rewrite to me

I'm going to ask @sensibleworld if it is fine to keep this request open

**Comment by mikkelsiggaard** [comment]
@tagliala 

Yes, I was afraid a nearly complete rewrite would be necessary, but wasn't sure. 

Hope you will consider it :)

**Comment by sensibleworld** [comment]
Thanks for the comment @mikkelsiggaard! Let's go ahead and leave this request open, @tagliala, and see how much community support there is for it. I'm not opposed to create duotone versions of our `light` and `regular` icons, but it is a fairly large task :)

**Comment by sensibleworld** [comment]
This now exists :) https://fontawesome.com/search?ip=duotone&o=r
